internal/operators/lvm/common.go

```diff
@@ -0,0 +1,62 @@
+package lvm
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+type baseHardware struct {
+	CPUCores int64 `json:"cpu_cores"`
+	RAMMib   int64 `json:"ram_mib"`
+}
+
+type hardwareRequirement struct {
+	Version string       `json:"version,omitempty"`
+	Master  baseHardware `json:"master"`
+	Worker  baseHardware `json:"worker"`
+	Sno     baseHardware `json:"sno"`
+}
+
+const (
+	envRequirementsKey string = "HW_VALIDATOR_REQUIREMENTS"
+	urlRequirement     string = "https://raw.githubusercontent.com/openshift/assisted-service/master/data/default_hw_requirements.json"
+)
+
+//TODO: move to common and use it for all operators
+
+type pageDownloader func(s string) (*http.Response, error)
+
+func downloadFromUrl(pageDownloadFunc pageDownloader) (*http.Response, error) {
+	return pageDownloadFunc(urlRequirement)
+}
+
+func get_page(url string) (*http.Response, error) {
+	return http.Get(url)
+}
+
+func getBaseRequiemnts(pageDownloadFunc pageDownloader) (hardwareRequirement, error) {
```

Comment:
Why are we downloading the requirements from a URL?

Reply:
This is for local deployment and/or testing.

The service is realying on env var only. we can remove it if you think we can ignore it

Reply:
It looks weird.  Why not just use env var?